spec/features/auth_feature_spec.rb

```diff
@@ -9,7 +10,8 @@
     fill_in "Password", :with => "test"
     fill_in "Confirm password", :with => "test"
     check "user_tos"
-    UserMailer.should_receive(:deliver_activation_instructions)
+    UserMailer.should_receive(:activation_instructions).and_return(email)
```

Comment:
I can never keep up with what version of rspec we're on but we should use the `allow(UserMailer).to_receive` syntax instead if it's available